ramls/eventDescriptor.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Event Descriptor data model",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "eventType": {
+      "description": "Unique Event type, human-readable String, not UUID",
+      "type": "string"
+    },
+    "description" : {
+      "description": "Detailed description of the event type",
+      "type": "string"
+    },
+    "eventTTL": {
+      "description": "Default time-to-live (TTL) for events of this type in minutes",
+      "type": "integer"
+    },
+    "signed": {
```

Comment:
Is this needed at the moment? Could it be deferred until the work where events are to be signed?

Reply:
We can put a default value 'false' and leave it at that for now.

Reply:
If there is no behaviour associated with it, I think it is clearer to leave it out of an initial version. 

What benefit do we get from adding it early?

FOLIO has had to break compatibility a number of times due to introducing ideas too early before their use, and then realised they don’t quite fit. With a hub like interface such as this one, that change could be expensive.

Reply:
Well, this field is not required and has a default value, so it shouldn't break anything. @TarasSpashchenko, what do you think? Would it be OK to remove this field, or should we leave it as is for now?

Reply:
let's leave it as is for now. This possibility is mentioned in the RFC, and it is stated that it is to be implemented in further versions.

Reply:
@TarasSpashchenko Ok, I was only sharing my concern about premature commitment to design (which has caught many modules in FOLIO out before)

Reply:
Thank you @marcjohnson-kint, do you think we could move on with this PR? I think we addressed all the other concerns.

Reply:
@KaterynaSenchenko That is entirely up to you and @TarasSpashchenko 

I was only providing my thoughts as requested (and those conversations have all concluded).

Reply:
Thanks, @TarasSpashchenko, I will merge this PR then, if there is no objections.